src/en/credentials.md

```diff
@@ -239,13 +261,13 @@ aws     bob*, carol
 google  wayne
 ```
 
-The asterisk '*' denotes the default credential, which will be used for the
-named cloud unless another is specified.
+The asterisk denotes the default credential, which will be used for the named
```

Comment:
Not too sure what 'named' here means... It really is 'the default for that cloud' I think.

Reply:
Will reword.